pkg/config/remote/service/service.go

```diff
@@ -186,7 +186,12 @@ func NewService() (*Service, error) {
 	}
 	clock := clock.New()
 
-	clientsCacheBypassLimit := config.Datadog.GetInt("remote_configuration.clients.cache_bypass_limit")
+	var clientsCacheBypassLimit int
+	if config.Datadog.IsSet("remote_configuration.clients.cache_bypass_limit") {
+		clientsCacheBypassLimit = config.Datadog.GetInt("remote_configuration.clients.cache_bypass_limit")
+	} else {
+		clientsCacheBypassLimit = defaultCacheBypassLimit
+	}
```

Comment:
Why not set the default directly in the config ?

Reply:
It was to keep our constants in the same place, but it's simpler to add a default, we do this for other things. Fixed!